aws/sqs_test.go

```diff
@@ -0,0 +1,83 @@
+package aws
+
+import (
+	"testing"
+	"time"
+
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sqs"
+	"github.com/gruntwork-io/cloud-nuke/util"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func createTestQueue(t *testing.T, session *session.Session, name string) (*string, error) {
```

Comment:
NIT: This method never returns an error; it just fails the test. Consider returning solely the queue URL to simplify the API.

Reply:
great! Done!